plugins/GSdx/Renderers/DXCommon/GSDeviceDX.h

```diff
@@ -39,11 +39,15 @@ class GSDeviceDX : public GSDevice
 		GSVector4 VertexOffset;
 		GSVector4 Texture_Scale_Offset;
 
+		GSVector2i DepthMask;
+
 		struct VSConstantBuffer()
 		{
 			VertexScale = GSVector4::zero();
 			VertexOffset = GSVector4::zero();
 			Texture_Scale_Offset = GSVector4::zero();
+
+			DepthMask = GSVector2i(0);
```

Comment:
I'm not sure if I should update the cb Update below.

Reply:
I think so, otherwise the update does not take into account the new member of the struct